### PR TITLE
Refactor get_authors.rb

### PR DESCRIPTION
**Pull request by jackorp**
Use #collect instead of #each.
Create a method get_authors to get last 5 authors and their email from file and test that method.

### CODE REVIEW

File: get_authors.rb [hunk 1]
```diff
@@ -12,15 +17,9 @@ def md_center(text)
...
-  author_mail = {}
-  author_md = []
-  command = 'git log -5 --pretty=format:"%an;%ae" '
-  command_out = %x(#{command}#{f}).split(/[;\n]/)
-  author_mail = command_out.each_slice(2).to_h.sort
-
-  author_mail.each do |a, e|
-    author_md << md_mailto(a, e)
-  end
+  author_mail = get_authors(f).each_slice(2).to_h.sort
```
Comment (pvalena) [inline get_authors.rb:20]: I do still see some `magic` in here. As this is only refactoring, and the line was not changed, I do not object.

**Comment by pvalena** [comment]
The rest seems fine, but I'll leave the final review to @voxik.

Note, that the CI failure persists:
```
undefined method `to_h' for #<Enumerator: ["Your Name", "you@example.com"]:each_slice(2)> (NoMethodError)
```

**Comment by voxik** [comment]
I see this as a step in right direction, so if that works for you locally, I'd appreciate if you can merge.

**Comment by pvalena** [comment]
Thanks!

**Pull request merged**